python/mlcroissant/mlcroissant/_src/structure_graph/nodes/file_object.py

```diff
@@ -53,9 +53,14 @@ class FileObject(Node):
         input_types=[SDO.Text],
         url=SDO.description,
     )
-    encoding_format: str | None = mlc_dataclasses.jsonld_field(
+    encoding_format: list[str] | None = mlc_dataclasses.jsonld_field(
```

Comment:
It would become `encoding_formats` (plural)

Reply:
Yeah good point!